estate/models/estate_property.py

```diff
@@ -0,0 +1,45 @@
+from dateutil.relativedelta import relativedelta
+from odoo import fields, models
+
+#Class of EstateProperty to define fields of database table
+class EstateProperty(models.Model):
+    _name="estate.property"
+    _description="Real Estate Properties"
+
+    name=fields.Char(string='Name',required = True)
+    description=fields.Text(string='Description')
+    postcode=fields.Char(string='Postcode')
+    date_availability=fields.Date(string='Available From',copy = False,default = fields.Date.add(fields.Date.today()+ relativedelta(months=3)))
```

Comment:
```suggestion
    date_availability= fields.Date(string='Available From', copy=False, default=fields.Date.add(fields.Date.today() + relativedelta(months=3)))
```

Please add spaces in the correct places and follow the same formatting for the rest of the code.

Also, could you clarify the purpose of adding the default here?

Reply:
Noted. will take care about it from now.

we can add default value to fields by using default attribute. And as per requirement of chapter default date availability date should be  in 3 months